rts/Lua/LuaOpenGL.h

```diff
@@ -4,9 +4,14 @@
 #define LUA_GL_H
 
 #include <vector>
+#include <string>
+#include <unordered_set>
 
+#include "Rendering/GL/myGL.h"
 #include "Lua/LuaHandle.h"
 
+#include "Rendering/GL/VBO.h"
```

Comment:
are these Rendering includes necessary to be in the header? whenever possible move them to the .cpp so we reduce compilation time